server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -203,6 +203,7 @@ data:
             "serverUrl": "{{ .Values.alfred.externalUrl }}",
             "blobStorageUrl": "{{ .Values.historian.externalUrl }}",
             "internalBlobStorageUrl": "{{ .Values.historian.internalUrl }}",
+            "gitrestUrl": "{{ .Values.gitrest.internalUrl }}",
```

Comment:
You will also need to add this to https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/kubernetes/routerlicious/values.yaml.

Or as I suggested in the other comment, add a `storageUrl` to the `storage` section in the r11s Fluid config map.

Reply:
Done. :)